eslint.config.mjs

```diff
@@ -74,6 +74,7 @@ export default tsEslint.config({
     "sonarjs/function-return-type": "off",
     "sonarjs/no-misleading-array-reverse": "off",
     "sonarjs/slow-regex": "off",
+    "sonarjs/no-nested-template-literals": "off",
```

Comment:
Perhaps it makes sense to also make adjustments to the ts-template

Reply:
I think it's a sonar js bug, basically happens when you nest like `\`${myVar}\`` which is wrong because escaped ` should not count as nested literal.